bears/python/PythonPackageInitBear.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from coalib.results.Result import Result
+from coalib.bears.GlobalBear import GlobalBear
+
+
+class PythonPackageInitBear(GlobalBear):
+    """
+    Looks for missing __init__.py files in directories containing python files.
+    """
+
+    LANGUAGES = {'Python', 'Python 3', 'Python 2'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    def run(self):
+        files = sorted(self.file_dict)
+        dirs = set()
+
+        for file in files:
+            dir = os.path.split(file)[0]
+            dirs.add(dir)
+
+        for dir in dirs:
+            if not os.path.join(dir, '__init__.py') in files:
+                yield Result(self,
+                             'Directory {} does not contain __init__.py file'
```

Comment:
I wouldn't speak about the directory but point out that the user should have a file with the givn path. Also ideally we have the relative path to `self.get_config_dir()`, I think coala does the same in ConsoleInteraction.py